core/src/test/resources/view/ValidViewMetadata.json

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
Looks like puffin files are in path `org/apache/iceberg/puffin/v1`, we should follow the same pattern